documentation/tutorials/batch.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: documentation
+menu: tut_batch
+subtitle: 'Tutorial: Batch'
+redirect_from:
+- /getStarted/batch/
+- /documentation/getstarted/batch/
+- /documentation/getstarted/advanced/batch/
+---
+# Tutorial: Batching
+{% include teams.html %}
+
+This brief tutorial will teach you **how to use the flyway batch command**.
+
+## Introduction
+
+Batching allows multiple `INSERT`, `UPDATE`, `DELETE`, `MERGE`, and `UPSERT` statements in a migration script to be sent over the wire to to database server in one go.
```

Comment:
`over the wire to to` I think ought to be `to the`?

Reply:
7396b4acb087ab37188d30a069a017c95164a7c2